ui/src/app/shared/header/chips/enum-selection.component.ts

```diff
@@ -0,0 +1,32 @@
+import {Component, EventEmitter, Input, OnInit, Output} from "@angular/core";
+
+@Component({
+  selector: 'jm-enum-selection',
+  templateUrl: './enum-selection.component.html',
+})
+export class EnumSelectionComponent implements OnInit {
```

Comment:
Should the `Status` chip use this?

Reply:
Can you clarify?  The `Status` chip is a special case where there can be multiple values (e.g. checkboxes in the UI).  All other fields and labels have only one value, so ones with set values to choose from should use this filter chip type (e.g. radio buttons in UI).

Reply:
Good point - forgot about this change. Ignore